build.gradle

```diff
@@ -466,20 +466,20 @@ def connectPkgs = ['connect:api', 'connect:runtime', 'connect:transforms', 'conn
 def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:examples'] + connectPkgs
 
 tasks.create(name: "jarConnect", dependsOn: connectPkgs.collect { it + ":jar" }) {}
-tasks.create(name: "jarAll", dependsOn: ['jar_core_2_10', 'jar_core_2_11'] + pkgs.collect { it + ":jar" }) { }
+tasks.create(name: "jarAll", dependsOn: ['jar_core_2_11'] + pkgs.collect { it + ":jar" }) { }
```

Comment:
That's a good suggestion. Added a commit that attempts this. Let me know what you think.